### PR TITLE
Add cypress dashboard in CI/CD sub generator

### PR DESCRIPTION
**Pull request by avdev4j**
close https://github.com/jhipster/generator-jhipster/issues/13716

- [X] Github actions
- [x] Azure pipelines
- [x] Circle CI
- [x] Travis
- [x] Jenkins?
- [x] tests
- [x] Documentation -> https://github.com/jhipster/jhipster.github.io/blob/main/pages/setting_up_ci.md

Please make sure the below checklist is followed for Pull Requests.

- [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]
Can you add npm test too plz?

**Comment by pascalgrimaud** [comment]
Then, we need documentation too, to not forget for the next release

**Comment by FanchGadjo** [comment]
:heavy_check_mark: I have ran a generation for GitHub Actions with an app without entities.

https://github.com/FanchGadjo/test-jh-13716
https://dashboard.cypress.io/projects/v9ab2j/runs/2/overview

**Comment by FanchGadjo** [comment]
Is something preventing this dev to be done on gitlab-ci as well ?

**Comment by avdev4j** [comment]
@pascalgrimaud I have added documentation and tests in the TODO list

@FanchGadjo It has been done for Gitlab too. Only Jenkins doesn't contain that.

File: generators/client/templates/angular/package.json.ejs [hunk 1]
```diff
@@ -154,7 +154,7 @@
     <%_ } _%>
     <%_ if (cypressTests) { _%>
     "e2e:cypress:headless": "<%= clientPackageManager %> run e2e:cypress -- --headless",
-    "e2e:cypress": "cypress run --browser chrome",
+    "e2e:cypress": "cypress run --browser chrome --record ${CYPRESS_ENABLE_RECORD:-false}",
```
Comment (mshima) [inline generators/client/templates/angular/package.json.ejs:157]: Is this portable?
Scripts starting with `ci:` are not portable.
But for this script IMO it should be.

Comment (avdev4j) [reply generators/client/templates/angular/package.json.ejs:157]: good question, I don't know. But I did not succeed on use arguments with the current "ci:" scripts

Comment (mshima) [reply generators/client/templates/angular/package.json.ejs:157]: See original bug about this https://github.com/jhipster/generator-jhipster/issues/12805.
Have you tried `concurrently -k -s first "npm run ci:e2e:server:start" "npm run e2e:headless -- --record ${CYPRESS_ENABLE_RECORD:-false}"`

Comment (avdev4j) [reply generators/client/templates/angular/package.json.ejs:157]: I have tested it but it doesn't work (If I remember I have already tested this before).

You can check logs here:
https://github.com/avdev4j/jhipster-cypress-dashboard/runs/1810120628?check_suite_focus=true

Capture d’écran 2021-02-01 à 22 50 58

**Comment by pascalgrimaud** [comment]
@avdev4j : can you solve the conflict plz ? so it can be merged

**Pull request merged**